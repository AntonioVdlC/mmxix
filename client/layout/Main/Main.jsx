import React from "react";
import PropTypes from "prop-types";

import styles from "./Main.css";

const Main = ({ children }) => <main className={styles.main}>{children}</main>;

Main.propTypes = {
  children: PropTypes.node.isRequired
};

export default Main;
