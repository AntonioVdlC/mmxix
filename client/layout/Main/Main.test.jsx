import React from "react";
import renderer from "react-test-renderer";

import Main from "./Main";

describe("Main", () => {
  it("should render correctly", () => {
    const tree = renderer.create(<Main>Helsinki is great!</Main>).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
