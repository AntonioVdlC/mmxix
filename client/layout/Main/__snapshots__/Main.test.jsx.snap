// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Main should render correctly 1`] = `
<main
  className="main"
>
  Helsinki is great!
</main>
`;
